### PR TITLE
Added Semantic UI support; improved mobile layout

### LINKED ISSUE #2: A picture of a persona in persona page and maybe a css framework for a nicer ui
Hi,

it would be nice if you could add images of the persona, if there any exists.
And it will also be nice if you can use a css framework like bootstrap to let the page look a little nicer.

### PR DESCRIPTION
**Pull request by vlxs**
Partially closes #2

### CODE REVIEW

File: src/PersonaListController.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * Created by Chin on 08-Apr-17.
  */
 var PersonaListController = (function () {
-    function PersonaListController($scope) {
+    function PersonaListController($scope, $timeout) {
```
Comment (chinhodado) [inline src/PersonaListController.js]: Is $timeout used?

File: src/App.js [hunk 2]
```diff
@@ -2,6 +2,7 @@
 ///<reference path="PersonaListController.ts"/>
 ///<reference path="SkillListController.ts"/>
 var myModule = angular.module('myModule', ['ngRoute']);
+myModule.directive('stickyTable', StickyTableDirective);
```
Comment (chinhodado) [inline src/App.js]: I don't think this needs to be added here?

**Comment by chinhodado** [comment]
Thanks for this! Looks like there's a couple of places that can be cleaned up but that can be done later. Also I'll need to write the changes in JS in Typescript later (most of the JS files in the project are compiled from the corresponding Typescript file).

File: src/PersonaListController.js (hunk 1 above)
Comment (vlxs) [inline src/PersonaListController.js]: Sorry, only saw this now. It's not used, it was part of an early change that I eventually removed; feel free to delete it.

**Pull request merged**